packages/client/lib/rpc/modules/consensus.ts

```diff
@@ -0,0 +1,108 @@
+import { middleware, validators } from '../validation'
+import type { Chain } from '../../blockchain'
+import type { EthereumClient } from '../..'
+import type { EthereumService } from '../../service'
+import type { EthProtocol } from '../../net/protocol'
+import type VM from '@ethereumjs/vm'
+
+/**
+ * eth_* RPC module
+ * @memberof module:rpc/modules
+ */
+export class Consensus {
+  private _chain: Chain
+  private _vm: VM | undefined
+  public ethVersion: number
+
+  /**
+   * Create eth_* RPC module
+   * @param client Client to which the module binds
+   */
+  constructor(client: EthereumClient) {
+    const service = client.services.find((s) => s.name === 'eth') as EthereumService
+    this._chain = service.chain
+    this._vm = (service.synchronizer as any)?.execution?.vm
+
+    const ethProtocol = service.protocols.find((p) => p.name === 'eth') as EthProtocol
+    this.ethVersion = Math.max(...ethProtocol.versions)
```

Comment:
you won't need this since it was just for `eth_protocolVersion`
```suggestion
```

Reply:
Thanks resolved. Also removed the `ethVersion` property from the class declaration